summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/batch/BatchedStore.scala

```diff
@@ -204,26 +211,24 @@ trait BatchedStore[K, V] extends scalding.Store[K, V] { self =>
         .toTypedPipe
     }
 
-    /**
```

Comment:
is this just a refactoring or does it fix serialization issues as well?

Reply:
it does! see how tricky it is! without this change, you serialize the whole store to get flatOpt, when really we don't need to grab that function for two simple use cases. Not even sure there is no flatten on Option still in scala, but in any case, flatten is not the best call in my view.